### ISSUE TITLE
Full version of code

### ISSUE THREAD

**Issue by watermelon097** [yuggiehk/CaRe-Ego#2]
Hi,

I was reading the paper that you have uploaded and I have found it really interesting. I wanted to inquire if you plan to release the full version of the ORMNET code, including the checkpoint. If so, could you provide an estimated timeline for when this will be available?

Thank you for your time.

Best regards.

**Comment by yuggiehk** [comment]
Hi! Thanks for your attention to this work. The training code for this work has been released on this repo, and the inference code and checkpoint will be released soon:)

**Comment by watermelon097** [comment]
> Hi! Thanks for your attention to this work. The training code for this work has been released on this repo, and the inference code and checkpoint will be released soon:)

Hi,
I have check the current code, but it is a bit hard to follow. Will you upload some training script before you release the whole project?

Best.

**Comment by yuggiehk** [comment]
Thanks for your suggestions.

Yes, I will upload instructions about training and inference with the whole project. 

Best.
